include/swift/RemoteInspection/RuntimeHeaders/llvm/ADT/BitmaskEnum.h

```diff
@@ -89,6 +89,8 @@
   /* Force a semicolon at the end of this macro. */                            \
   using ::llvm::BitmaskEnumDetail::operator^=
 
+inline namespace __swift {
+inline namespace __runtime {
```

Comment:
This seems like playing with different ODR.  This is the namespace that we use in the runtime, so that would now cause a possible issue.  Can we use something like `__swift::__inspection`?

Reply:
Done